lib/bootstrap.js

```diff
@@ -6,17 +6,33 @@ require('./utils/process-log');
 
 const isFlagPresent = (args, flag) => args.find(arg => [flag, `--${flag}`].includes(arg));
 const normalizeFlags = (args, cmd) => args.slice(2).filter(arg => arg.indexOf('--') < 0 && arg !== cmd.name && arg !== cmd.alias);
+const validFlags = [...core.map(obj => obj.name), ...commands.map(obj => obj.name)]
 
 const isCommandUsed = commands =>
     commands.find(cmd => {
         return process.argv.includes(cmd.name) || process.argv.includes(cmd.alias);
     });
 
+const checkValidFlag = () => {
+    let status = true
+    process.argv.slice(2).forEach(flag => {
```

Comment:
please suggest if you have refactoring for this in mind

Reply:
```js
const checkFlagIsValid = () => {
  return !process.argv.some(flag => {
    return !validFlags.includes(flag);    
  });
};
```
This is what I have in mind (seems a bit confusing due to the negations).

Reply:
I guess you could get rid of the `status` variable and directly return a boolean value.